### ISSUE TITLE
Update Hosted Che to the latest `7.3.1` release

### ISSUE THREAD

**Issue by ibuziuk** [eclipse-che/che#14656]
Need to update Hosted Che to the latest `7.3.0` release
Draft PR - https://github.com/redhat-developer/rh-che/pull/1626
Dependent on https://github.com/eclipse/che/issues/14896

**Comment by amisevsk** [comment]
Blocked by https://github.com/eclipse/che/issues/14749

**Comment by ibuziuk** [comment]
Related PRs:
- https://github.com/openshiftio/saas-openshiftio/pull/1340
- https://github.com/openshiftio/saas-openshiftio/pull/1341
- https://github.com/openshiftio/saas-openshiftio/pull/1342

**Comment by ibuziuk** [comment]
Hosted che update is postponed due to https://github.com/eclipse/che/issues/14944
we are going to update prod with `7.3.1` where the regression is fixed

**Comment by ibuziuk** [comment]
Related PRs for prod update to `7.3.1` 
- https://github.com/openshiftio/saas-openshiftio/pull/1343
- https://github.com/openshiftio/saas-openshiftio/pull/1344
- https://github.com/openshiftio/saas-openshiftio/pull/1345

Waiting for QA approval before promotion to production

**Comment by ibuziuk** [comment]
Hosted Che has been updated to the latest 7.3.1 version. Closing
